scripts/initial_release_prep.py

```diff
@@ -0,0 +1,312 @@
+AUTH_FILEPATH = os.path.join(os.getcwd(), 'sheets.googleapis.com-python.json')
+
+
+def get_job_details_for_current_release(
+        job_details, job_name_header, month_header, author_email_header,
+        author_name_header, instruction_doc_header):
+    """Returns a list of job names for current release from a list
+    of job details.
+
+    Args:
+        job_details: list(dict). List of dictionary of job details.
+            The keys of each dict are the headers from the jobs
+            spreadsheet.
+        job_name_header: string. The header used for job names.
+        month_header: string. The header used for month of the release.
+        author_email_header: string. The header used for email address of
+            job author.
+        author_name_header: string. The header used for name of
+            job author.
+        instruction_doc_header: string. The header used for instruction
+            doc for running the job.
+
+    Returns:
+        list(dict). The list of job details for current release.
+    """
+    job_details_for_current_release = []
+    # First three letters of the release month. Eg: Nov, Dec etc.
+    current_release_month = datetime.datetime.utcnow().strftime('%h')
+    for job_detail in job_details:
+        if current_release_month in job_detail[month_header]:
+            job_details_for_current_release.append({
+                'job_name': job_detail[job_name_header],
+                'author_email': job_detail[author_email_header],
+                'author_name': job_detail[author_name_header],
+                'instruction_doc': job_detail[instruction_doc_header]
+                })
+    return job_details_for_current_release
+
+
+def get_mail_message_template(job_details):
+    """Returns a mail template with the details for running jobs for release
+    on backup server.
+
+    Args:
+        job_details: list(dict). List of dictionary of job details. The dict
+            has the following keys: job_name, author_email, author_name,
+            instruction_doc.
+
+    Returns:
+        string. The mail message template.
+    """
+    job_names_with_instructions_and_author = ''
+
+    for job_detail in job_details:
+        job_names_with_instructions_and_author += (
+            '%s (Instructions: %s) (Author: %s)\n' % (
+                job_detail['job_name'], job_detail['instruction_doc'],
+                job_detail['author_name']))
+    mail_message_template = (
+        'Hi Sean,\n\n'
+        'You will need to run these jobs on the backup server:\n\n'
+        '%s\n'
+        'The specific instructions for jobs are linked with them. '
+        'The general instructions are as follows:\n\n'
+        '1. Login as admin\n'
+        '2. Navigate to the admin panel and then the jobs tab\n'
+        '3. Run the above jobs\n'
+        '4. In case of failure/success, please send the output logs for '
+        'the job to me and the job author:\n\n'
```

Comment:
```suggestion
        'the job to me and %s:\n\n'
```
Add the authors name as I think we have that data directly right? You can also maybe add the author's email id.

Reply:
The author name is mentioned in the list of the jobs ([L109](https://github.com/oppia/oppia/pull/8005/files/60b8dad47d0d646f925dce41ea75a405c9d925c3#diff-5ec19cb5334ecdb7f1dada588baf5b5eR109), [L116](https://github.com/oppia/oppia/pull/8005/files/60b8dad47d0d646f925dce41ea75a405c9d925c3#diff-5ec19cb5334ecdb7f1dada588baf5b5eR116)) already. So, should we repeat it here?

Reply:
Makes it easier for Sean😅

Reply:
That's a good point 😛 

Updated!